makefiles/shared.mk

```diff
@@ -14,8 +14,6 @@ PATCH_ID=$(shell if test -f "${PATCH_FILE}"; then git patch-id < ${PATCH_FILE} |
 PATCH_CLAUSE=$(shell if [ ! -z "${PATCH_ID}" ]; then echo --patch "${PATCH_ID}"; fi)
 
 BUNDLE_PATH=${REPO_DIR}/bundle.zip
-RSTSPEC_FLAG=--rstspec=https://raw.githubusercontent.com/mongodb/snooty-parser/latest/snooty/rstspec.toml
```

Comment:
We should also remove the `${RSTSPEC_FLAG}` references in makefiles that override the `next-gen-parse` rule. I count 12 of them :(

* `Makefile.docs-atlas-cli`
* `Makefile.docs-atlas-operator`
* `Makefile.docs-landing`
* `Makefile.docs-laravel`
* `Makefile.mongocli`
* `Makefile.mongodb-internal-base`
* `Makefile.docs-mongodb-internal`
* `Makefile.mongoid`
* `Makefile.docs-php-library`
* `Makefile.docs-ruby`
* `Makefile.docs`
* `Makefile.docs-stage`

Reply:
great catch! removed from these makefiles